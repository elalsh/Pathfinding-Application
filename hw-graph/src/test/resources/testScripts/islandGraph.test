#Tests island graphs, graphs with most of the vertexes not having neighbors

# Create a graph
CreateGraph graph1

# Add nodes A, B, C, D, E, F, G
AddNode graph1 A
AddNode graph1 B
AddNode graph1 C
AddNode graph1 D
AddNode graph1 E
AddNode graph1 F
AddNode graph1 G

# add an edge from A -> B and C -> D
AddEdge graph1 A B edgeAB
AddEdge graph1 C D edgeCD

# Print the nodes in the graph and the outgoing edges for src nodes to dest nodes
ListNodes graph1
ListChildren graph1 A
ListChildren graph1 B
ListChildren graph1 C
ListChildren graph1 D
ListChildren graph1 E
ListChildren graph1 F
ListChildren graph1 G
