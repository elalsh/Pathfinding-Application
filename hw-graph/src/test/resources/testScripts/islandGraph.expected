#Tests island graphs, graphs with most of the vertexes not having neighbors

# Create a graph
created graph graph1

# Add nodes A, B, C, D, E, F, G
added node A to graph1
added node B to graph1
added node C to graph1
added node D to graph1
added node E to graph1
added node F to graph1
added node G to graph1

# add an edge from A -> B and C -> D
added edge edgeAB from A to B in graph1
added edge edgeCD from C to D in graph1

# Print the nodes in the graph and the outgoing edges for src nodes to dest nodes
graph1 contains: A B C D E F G
the children of A in graph1 are: B(edgeAB)
the children of B in graph1 are:
the children of C in graph1 are: D(edgeCD)
the children of D in graph1 are:
the children of E in graph1 are:
the children of F in graph1 are:
the children of G in graph1 are:

